index.html

```diff
@@ -0,0 +1,452 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta name="description" content="Cusana Insights is an intelligence platform for marketers and salespeople to better understand customers and measure their KPIs on social media.">
+      <meta http-equiv="refresh" content="3600;url=https://davidobi.github.io/CusanaInsights/">
+      <meta name="keywords" content="Cusana-Insights,Sales,Marketers,Social-media,Insights,Analytics">
+      <meta name="author" content="Cusana-Insights">
+      <meta name="robots" content="index,follow">
+      <meta name="copyright" content="Cusana Insights is a startup from Meltwater Entrepreneur School of Technology in Accra Ghana which is a registered Trademark.">
+      <meta name="DC.title" content="Cusana Insights -- An intelligence platform that helps marketers & sales Experts convert customers to friends.">
+      <title>Cusana Insights </title>
+      <link href="css/bootstrap.min.css" rel="stylesheet">
+      <link href="css/animate.min.css" rel="stylesheet"> 
+      <link href="css/font-awesome.min.css" rel="stylesheet">
+      <link href="css/lightbox.css" rel="stylesheet">
+      <link href="css/main.css" rel="stylesheet">
+      <link id="css-preset" href="css/presets/preset1.css" rel="stylesheet">
+      <link href="css/responsive.css" rel="stylesheet">
+
+      <!--[if lt IE 9]>
+        <script src="js/html5shiv.js"></script>
+        <script src="js/respond.min.js"></script>
+      <![endif]-->
+      
+      <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700' rel='stylesheet' type='text/css'>
+      <link rel="shortcut icon" href="images/drop124.png">
+      
+    </head><!--/head-->
+
+    <body>
+
+      <!--.preloader-->
+      <div class="preloader"> <i class="fa fa-circle-o-notch fa-spin"></i></div>
+      <!--/.preloader-->
+
+      <header id="home">
+        <div id="home-slider" class="carousel slide carousel-fade" data-ride="carousel">
+          <div class="carousel-inner">
+            <div class="item active" style="background-image: url(images/slider/future_health.jpg)">
+              <div class="caption">
+                <h1 class="animated fadeInLeftBig">Welcome to <span>Cusana Insights</span></h1>
+                <p class="animated fadeInRightBig">Understand your audience</p>
+
+                <a data-scroll class="btn btn-start animated fadeInUpBig" href="loginpage/login.html">Try now</a>
+              </div>
+            </div>
+            <div class="item" style="background-image: url(images/slider/family.jpg)">
+              <div class="caption">
+
+                <h1 class="animated fadeInLeftBig">Psychographic <span>and interest graphs</span></h1>
```

Comment:
Please, consider that the h1 tag could be used one time on your webpage. All other headings should be hx, aka h2, h3, h4, etc.